form.js

```diff
@@ -0,0 +1,32 @@
+var writeUsButton = document.querySelector(".writeus-button");
+var modalForm = document.querySelector(".modal-write");
+var fio = modalForm.querySelector("[name=user-name]");
+var formClose = document.querySelector(".write-close");
+var form = modalForm.querySelector("form");
+var email = modalForm.querySelector("[name=user-email]");
+
+         
+writeUsButton.addEventListener("click", function(evt) {
+    evt.preventDefault();
+    modalForm.classList.add("form-show");
+    fio.focus();
+});
+formClose.addEventListener("click", function(evt) {
+    evt.preventDefault();
```

Comment:
same question as above. I just want you to understand for this preventDefaults are needed

Reply:
yes, here nothing to preventDefault. Removed evt.